lib/commands/actions.js

```diff
@@ -43,6 +43,15 @@ commands.setOrientation = async function (orientation) {
   return await this.bootstrap.sendAction("orientation", params);
 };
 
+commands.setEmuOrientation = async function (adb, orientation) {
+  orientation = orientation.toUpperCase();
+  let currentOrientation = await this.getOrientation();
+  if (currentOrientation !== orientation) {
+    await adb.rotate();
+    await this.setEmuOrientation(adb, orientation);
```

Comment:
Is this because more than one rotation may be needed? A comment might clarify.

Reply:
@imurchie I think I confuse this with ROTATION, does getOrientation() only returns `landscape` or `portarit` ?